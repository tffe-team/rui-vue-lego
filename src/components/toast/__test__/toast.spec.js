import $toast from '../index'

test('$toast', () => {
  const toast = $toast({type: 'info'})
  expect(toast.$el.outerHTML).toMatchSnapshot()
})
