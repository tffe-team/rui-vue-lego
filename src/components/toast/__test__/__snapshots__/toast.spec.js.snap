// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`$toast 1`] = `"<div class=\\"r-vue-lego-toast vue-lego-slide-in-down\\" style=\\"display: none;\\"><div class=\\"toast-content toast-content--info\\"><span class=\\"message\\"></span></div></div>"`;
